### ISSUE TITLE
Tempo Slider sets tempo verbatim, not percentage

### ISSUE THREAD

**Issue by RavenAmbree** [parulina/bardmacroplayer#20]
Environment:
Performing with: BardMacroPlayer v1.4
Creating/editing with: MidiEditor

I think the tempo slider sets tempo verbatim.
I've noticed this phenomenon with songs I've downloaded off reddit, as well as a few I've edited myself.
I will provide the ones I've edited myself right here: [OneDrive Clicky](https://1drv.ms/u/s!AorJWF3T6zXC6AzzrtplK2KME1va)

What I'm saying is this:
- 100% = 100bpm.
- 150% = 150bpm.

I came across this because I tried to sync the song in game with the midi player. Take my CC - RD song. The tempo change event in the midi is set for 148 bpm at time code 0. I set the slider in BMP to 150%, which should push the bpm of the song past 200. However, BMP only outpaced the midi player by a tiny little bit. I set the tempo slider in BMP to 140%, and it lagged behind the midi player.

In short: Midi tempo event set in file at 140bpm.
Is 100% supposed to be 140 and 110% be 154bpm?

**Comment by pixaline** [comment]
Hello, apologies for late reply, been busy.
The speed slider adjusts the speed of every note, not the tempo itself. I'll take this in to account for the next version, which will be an entirely new app. Thank you
